### ISSUE TITLE
Script returned with error: win-sfc-sf-corruption and win-chkdsk-fs-corruption

### ISSUE THREAD

**Issue by LeonardSchwenk** [Azure/repair-script-library#68]
Hello,

I just ran 
`az vm repair run -g xxx-rg -n xxxvm-1 --run-id win-chkdsk-fs-corruption --verbose`

as well as 

`az vm repair run -g xxx-rg -n xxxvm-1 --run-id  win-sfc-sf-corruption --verbose`

Both commands did not work and gave me the follwing error message.

The vm was still bootable and usable but had a disk error.

## error

>Running script on VM: xxxdvm-1
Log file is too large and has been cutoff at the start of file. Please locate the log file within the VM using the logFullpath to check full logs.

Script returned with error:

```
{
  "log_full_path": "",
  "logs": "",
  "message": "Script completed with errors.",
  "output": "",
  "resource_group": "xxx-rg",
  "script_status": "ERROR",
  "status": "SUCCESS",
  "vm_name": "xxxvm-1"
}
```

I cannot check the logs status is SUCCESS but i doubt that it worked because  `win-chkdsk-fs-corruption` took just 70 seconds for 512 GB disk.

Any help appicieated.

**Comment by rymccall** [comment]
Can you confirm your version of vm-repair with `az extension show --name vm-repair`? If it is `0.5.8`, can you try updating to `0.5.9` and retrying: `az extension update --name vm-repair`

If that fails, can you try downgrading to `0.5.7` and retrying:

```bash
az extension remove --name vm-repair
az extension add --name vm-repair --version 0.5.7
```

**Comment by LeonardSchwenk** [comment]
@rymccall 

Version is `0.5.9`

So should I downgrade to 0.5.7 ?

**Comment by rymccall** [comment]
Yes, try that out and confirm if successful, if so there may be an issue with 0.5.9 as well

**Comment by EdwinBernal1** [comment]
The actual version is 2.2.0 

I couldn't replicate the error if @LeonardSchwenk you could help us to validate if right now keep happening the error could you share it.

<img width="693" height="55" alt="Image" src="https://github.com/user-attachments/assets/48e536e3-1093-4feb-9644-acda4663d51a" /> 

<img width="2532" height="1035" alt="Image" src="https://github.com/user-attachments/assets/8f4078d5-515c-43a6-9e26-5f11628c4962" />
